eMPRess_GUI_version1.py

```diff
@@ -0,0 +1,354 @@
+# eMPRess_GUI_iteration5
+
+import tkinter as tk
+from tkinter import messagebox
+from pathlib import Path
+import numpy as np
+import matplotlib.pyplot as plt
+from matplotlib.backends.backend_tkagg import FigureCanvasTkAgg, NavigationToolbar2Tk
+from matplotlib.backend_bases import MouseEvent, key_press_handler
+import empress
+
+class App:
+
+    def __init__(self, master):
+        self.master = master
+        ### configures the master frame ###
```

Comment:
Please use one `#` for block comments: `# configures the master frame`.

Reply:
Done:)